src/TouristInfoCards.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+
+// getting whole object which was passed from `App()` in 'props' and then get separate values from that object (name, link, image)
+const TouristInfoCards = props => {
```

Comment:
Does this component represent all of the cards? Or is just 1 card? Based on this, do you think we should rename this component?